public/js/monitor.js

```diff
@@ -92,16 +92,22 @@ function handleFormSubmits(formEvent) {
     email = thisForm.email.value.trim();
     thisForm.email.value = email;
   }
+  const formClassList = thisForm.classList;
```

Comment:
This seems to work fine, but it reminds me when [we accidentally assigned an element's property value into a variable, manipulated the variable value, and forgot to assign the manipulated value back into the underlying element](https://github.com/mozilla/blurts-server/commit/85fd4e83c1a03b06143b6cc806f8974d676ca34a#diff-12528c522ce5f8325d894770704401fb).

To ensure the element's `classList` value is updated before submitting the form, can we add a line `thisForm.classList = formClassList` before `thisForm.submit()` ?